latest/Dockerfile.amd64

```diff
@@ -1,5 +1,5 @@
 # since we require a non-LTS release (which ships Qt 5.15 by default), we base the image directly on the ubuntu base image
-FROM ubuntu:21.04
+FROM ubuntu:22.04
```

Comment:
Will it be safe to update base to `22.04` for building desktop-client and running ctests?
As I have commented [here](https://github.com/owncloud-ci/client/pull/15#issuecomment-1227125737) ctests were running fine

CC @fmoc @TheOneRing

Reply:
But it would be great to have the current latest image as a backup tag

Reply:
Sure, 22.04 is supported.

Reply:
> But it would be great to have the current latest image as a backup tag


@fmoc Can we have a backup of the current latest image before merging this?
moving `owncloudci/client:latest` to something like `owncloudci/client:ubuntu-21.04`

Reply:
I have made a backup on my repo: https://hub.docker.com/r/sawjan/client/tags
If that works then this PR is ready to be reviewed and merged!